src/ngraph/pass/cse.hpp

```diff
@@ -26,6 +26,17 @@ namespace ngraph
     }
 }
 
+/// The Common Subexpression Elimination pass removes duplicate computations in a given computation graph.\n
+/// Two commutations are considered to be duplicate to each other if both applies same operation on same set of inputs,
```

Comment:
1. Please change to "computations".
2. both "apply"

Reply:
Done.